### PR TITLE
html: calculator

### PR DESCRIPTION
**Pull request by kristiyanpetkov**
In my PR i made an html calculator. I use external css file for the style and external javascript file for the calculation logic.

screenshot from 2016-05-09 15-46-14

### CODE REVIEW

File: src/com/clouway/calculator/calculator.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="display">
+        <input id="value" name="textfield" value="0" readonly size="33" class="text">
+    </div>
+
+    <div class="keys">
+
+        <p><input type="button" class="button" name="C" value="C" onclick="clearAll()">
+            <input type="button" class="button" value="CE" onclick="clearLastSymbol()">
+            <img src="casiologo.png"  style="width:150px; height:28px; padding-left: 20px">
+        </p>
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.html]: format the code:
```js
<p>
      <input type="button" class="button" name="C" value="C" onclick="clearAll()">
      <input type="button" class="button" value="CE" onclick="clearLastSymbol()">
      <img src="casiologo.png"  style="width:150px; height:28px; padding-left: 20px">
</p>
```

**Comment by EmilGeorgiev** [comment]
It is better to add pictures when make UI.

File: src/com/clouway/calculator/calculator.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+var enterOnce;
+function numInput(digit) {
+  var display = document.getElementById("value");
+  if (display.value === '0') {
+    display.value = '';
+    display.value += digit;
+  } else {
+    display.value += digit;
+  }
+}
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.js]: Refactor to:
```js
var display = document.getElementById("value");
if (display.value === '0') {
    display.value = '';
}

display.value += digit;
```

File: src/com/clouway/calculator/calculator.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<head>
+    <title>Online Calculator</title>
+    <link rel="stylesheet" href="calculator.css"/>
+    <script src="calculator.js"></script>
+</head>
+
+<body>
+<div class="container">
+    <div class="display">
+        <input id="value" name="textfield" value="0" readonly size="33" class="text">
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.html]: maybe ```id=...``` should be ```display``` or ```dashboard``` or ```screen```,  ```value``` is to common

File: src/com/clouway/calculator/calculator.html (hunk 1 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.html]: Fixed.

File: src/com/clouway/calculator/calculator.js (hunk 2 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.js]: Fixed.

File: src/com/clouway/calculator/calculator.html (hunk 3 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.html]: Renamed to ```screen```.

File: src/com/clouway/calculator/calculator.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            <input type="button" class="button" value="CE" onclick="clearLastSymbol()">
+            <img src="casiologo.png" style="width:150px; height:28px; padding-left: 20px">
+        </p>
+
+
+        <p><input type="button" class="button" value="7" onclick="numInput('7')">
+            <input type="button" class="button" value="8" onclick="numInput('8')">
+            <input type="button" class="button" value="9" onclick="numInput('9')">
+            <input type="button" class="button" value="/" onclick="setOperation('/')">
+        </p>
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.html]: .

File: src/com/clouway/calculator/calculator.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+var enterOnce;
+function numInput(digit) {
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.js]: by convention name of function should contains verb. Here ```numInput``` doesn't sound as name of function. Maybe ```enterNumber``` is more readable

File: src/com/clouway/calculator/calculator.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  var display = document.getElementById("value");
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.js]: value -> sreen

File: src/com/clouway/calculator/calculator.js [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+  if (display.value === '0') {
+    display.value = '';
+    display.value += digit;
+  }
+  display.value += digit;
+}
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.js]: I think that line 6 is unnecessary. Line 8 is same.

File: src/com/clouway/calculator/calculator.js [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+function decimalInput() {
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.js]: endterPoint

File: src/com/clouway/calculator/calculator.html (hunk 4 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.html]: fixed

File: src/com/clouway/calculator/calculator.js (hunk 5 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.js]: fixed

File: src/com/clouway/calculator/calculator.js (hunk 6 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.js]: fixed

File: src/com/clouway/calculator/calculator.js (hunk 7 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.js]: fixed

File: src/com/clouway/calculator/calculator.js (hunk 8 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.js]: fixed

File: src/com/clouway/calculator/calculator.js [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+    display.value = '';
+  }
+  display.value += digit;
+}
+
+function enterPoint() {
+  var display = document.getElementById("screen");
+  if (display.value.indexOf('.') === -1) {
+    display.value += '.';
+    enterOnce = false;
```
Comment (EmilGeorgiev) [inline src/com/clouway/calculator/calculator.js]: if here return function this will work ?

**Comment by EmilGeorgiev** [comment]
LGTM after fix coment

File: src/com/clouway/calculator/calculator.js (hunk 9 above)
Comment (kristiyanpetkov) [inline src/com/clouway/calculator/calculator.js]: yes, it will. return added.

**Pull request merged**